app/views/incomes/_income.html.erb

```diff
@@ -0,0 +1,9 @@
+<li><%= t("incomes.defaults.name") %>: <%= income.name %></li>
+<li><%= t("incomes.defaults.due_date") %>: <%= l(income.due_date, format: :default) %></li>
+<li><%= t("incomes.defaults.frequency") %>: <%= income.frequency %></li>
+<li><%= t("incomes.defaults.amount") %>: <%= number_to_currency(income.amount) %></li>
+<li><%= t("incomes.defaults.budget") %>: <%= income.budget.name %></li>
```

Comment:
Does budget still have a name? Can't remember

Reply:
It does, but I still think I will go with creating a budget with the user long term